### PR TITLE
[Py] Python wrappers for nGraph operations

### PR DESCRIPTION
**Pull request by arogowie-intel**
add:
- ng.absolute
- ng.acos
- ng.asin
- ng.atan

### CODE REVIEW

File: python/test/ngraph/test_ops_unary.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+import ngraph as ng
+
+
+def _get_runtime():
+    manager_name = pytest.config.getoption('backend', default='CPU')
+    return ng.runtime(manager_name=manager_name)
+
+
+def _run_unary_op_node(input_data, unary_op, as_node=True):
```
Comment (postrational) [inline python/test/ngraph/test_ops_unary.py]: If you use `as_node` you don't need to create or pass the parameter.
Let's split this into 2 functions, one to run the operation on a node, the other on numeric data.
You can then run both in each test, so no need to pass the extra `as_node` param.

Comment (arogowie-intel) [reply python/test/ngraph/test_ops_unary.py]: updated

File: python/test/ngraph/test_ops_unary.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    runtime = _get_runtime()
+    parameter_a = ng.parameter(input_data.shape, name='A', dtype=np.float32)
+    node = unary_op(parameter_a)
+    computation = runtime.computation(node, parameter_a)
+    return computation(input_data)
+
+
+def _run_unary_op_numeric_data(input_data, unary_op):
+    runtime = _get_runtime()
+    parameter_a = ng.parameter(input_data.shape, name='A', dtype=np.float32)
```
Comment (postrational) [inline python/test/ngraph/test_ops_unary.py]: `parameter` is redundant in this case

File: python/test/ngraph/test_ops_unary.py [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    assert np.allclose(result, expected)
+
+
+@pytest.mark.parametrize('ng_api_fn, numpy_fn, input_data', [
+    (ng.absolute, np.abs, -1 + np.random.rand(2, 3, 4) * 2),
+    (ng.absolute, np.abs, np.float32(-3)),
+    (ng.acos, np.arccos, -1 + np.random.rand(2, 3, 4) * 2),
+    (ng.acos, np.arccos, np.float32(-0.5)),
+])
+def test_unary_op_with_numeric_data(ng_api_fn, numpy_fn, input_data):
```
Comment (postrational) [inline python/test/ngraph/test_ops_unary.py]: We can combine these 2 tests into one.

File: python/test/ngraph/test_ops_unary.py (hunk 2 above)
Comment (arogowie-intel) [reply python/test/ngraph/test_ops_unary.py]: Fixed

File: python/test/ngraph/test_ops_unary.py (hunk 3 above)
Comment (arogowie-intel) [reply python/test/ngraph/test_ops_unary.py]: Fixed.

**Pull request merged**